app-nuxtjs-3-starter/README.md

```diff
@@ -16,27 +16,76 @@ npm install
 # pnpm
 pnpm install
 ```
```

Comment:
Since we have `yarn.lock`, we should remove the `npm install` and `pnpm install` commands.